### DISCUSSION TITLE
Cancel preview building/queued Git deployments when new Git triggered deployments start

### DISCUSSION THREAD

**Discussion by cjayyy** [vercel/community#6028] [Feature Question]
### Question

Is it possible to configure Vercel to automatically cancel all branch jobs when new job from same branch gets created? 
For example when I push 2 commits with some delay I only want the preview with the latest changes so all previous builds could be thrown away and save computation time.

**Comment by swarnava** [comment]
Hey, 

you can set `autoJobCancelation` to false in your `vercel.json` to achieve this: https://vercel.com/docs/projects/project-configuration/git-configuration#github.autojobcancelation

**Reply by cjayyy** [reply]
I **want to cancel** all previous running jobs when new job gets created so I should rather set `autoJobCancelation` to `true`, no?

**Reply by cjayyy** [reply]
@swarnava 
(sorry for mention, but the discussion isn't resolved from my POV and I cannot re-open it)
